### DISCUSSION TITLE
Enemy

### DISCUSSION THREAD

**Discussion by Dylanimal** [jeffaroundtheworld/2024gamecollab#54] [Polls]
How do we want the enemy to interact with the character? Right now it will go directly to the player and kill them after a couple second in proximity. What should change?

**Comment by kodriq** [comment]
the enemy cant go through walls, as plot wise it wouldn't make sense 
(they are "zombie" ish humans, so they have physical presences in the world)
I think, if possible, there will be an enemy that circles a certain area, like loops around a room, but if the player gets close enough they will turn and follow them instead
but if that's too complicated the other two options would still work fine, we might just need a boundary that the enemy cant cross so it doesn't follow you the entire level (since there are times you need to stop and talk to people etc)
